src/components/structures/MatrixChat.js

```diff
@@ -140,6 +141,7 @@ module.exports = React.createClass({
 
     componentWillMount: function() {
         this.favicon = new Favico({animation: 'none'});
+        this.guestCreds = null;
```

Comment:
I'd like to see this initialised in the `constructor`, with a comment as to what it means.